Scripts/swiftformat.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Set bash to Strict Mode (http://redsymbol.net/articles/unofficial-bash-strict-mode/)
+set -euo pipefail
+
+echo -e "👉 Running SwiftFormat Linting"
+
+echo -e "👉 Linting Sources..."
+mint run swiftformat --lint --config .swiftformat Sources --exclude **/Generated
+echo -e "👉 Linting Tests..."
+mint run swiftformat --lint --config .swiftformat Tests
+echo -e "👉 Linting Sample..."
+mint run swiftformat --lint --config .swiftformat Sample
+echo -e "👉 Linting DemoApp..."
+mint run swiftformat --lint --config .swiftformat DemoApp
+echo -e "👉 Linting Integration..."
+mint run swiftformat --lint --config .swiftformat Integration
+echo -e "👉 Linting DocsSnippets..."
+mint run swiftformat --lint --config .swiftformat-snippets DocsSnippets
```

Comment:
The name of this script is confusing. Maybe it should rather be `run_linter.sh` or something like that?

Reply:
Yes it is better to not confuse with the Swiftformat config

Reply:
✅